controllers/rhmi/rhmi_controller.go

```diff
@@ -842,30 +846,33 @@ func (r *RHMIReconciler) handleUninstall(installation *rhmiv1alpha1.RHMI, instal
 			if err != nil {
 				merr.Add(err)
 			}
-			return retryRequeue, merr
+			return merr
 		}
 
 		installation.SetFinalizers(resources.Remove(installation.GetFinalizers(), deletionFinalizer))
 
 		err = r.Update(context.TODO(), installation)
 		if err != nil {
 			merr.Add(err)
-			return retryRequeue, merr
+			return merr
 		}
 
 		if err := addon.UninstallOperator(context.TODO(), r.Client, installation); err != nil {
 			merr.Add(err)
-			return retryRequeue, merr
+			return merr
 		}
 
 		log.Info("uninstall completed")
-		return ctrl.Result{}, nil
```

Comment:
It looks to me like on this line, when the uninstallation is completed, the intention was to stop the RHMI Controller entirely. By sending an empty Result + nil we are effectively saying - you're done, don't ever run again. 
This makes me think why was there an update on 871 at all. IMO the last step of this entire flow should be empty result and nil as it was but maybe I'm missing something. With your changes, it seems that we are going to be req-queueing the RHMI controller indefinitely.